src/server/browserType.ts

```diff
@@ -39,11 +38,21 @@ export type LaunchOptions = BrowserArgOptions & {
   env?: {[key: string]: string} | undefined
 };
 
+export type ConnectOptions = {
+  wsEndpoint: string
+} & BrowserOptions;
+
+type BrowserOptions = {
```

Comment:
I am personally lost in all these types. For example, I don't understand why `BrowserOptions` include `slowMo`. Can we just inline all the one-offs?

Reply:
I inlined BrowserOptions, and changed the types we export.

BrowserArgsOptions: For default args method
LaunchOptions: For the launch and launchPersistent methods
LaunchServerOptions: For launchServer and _launchServer methods
ConnectOptions: For connect method

Hopefully things are easier to follow now!